src/main/scala/quasar/sbt/QuasarPlugin.scala

```diff
@@ -62,11 +64,13 @@ object QuasarPlugin extends AutoPlugin {
 
       quasarPluginDestinationFqcn := None,
 
+      quasarPluginSchedulerFqcn := None,
+
       quasarPluginExtraResolvers := Seq.empty,
 
       libraryDependencies := {
         val quasarDependencies =
-          if (srcFqcn.value.isDefined || dstFqcn.value.isDefined)
+          if (srcFqcn.value.isDefined || dstFqcn.value.isDefined || skdFqcn.value.isDefined)
             Seq(
               "com.precog" %% "quasar-connector" % quasarPluginQuasarVersion.value,
               "com.precog" %% "quasar-connector" % quasarPluginQuasarVersion.value % Test classifier "tests")
```

Comment:
Hmm, we're actually going to need different dependencies here. I'm not sure which module contains the scheduling things. I'll check @cryogenian's branch.